activerecord/lib/active_record/railties/databases.rake

```diff
@@ -19,7 +19,10 @@ db_namespace = namespace :db do
   end
 
   task load_config: :environment do
-    ActiveRecord::Base.configurations       = ActiveRecord::Tasks::DatabaseTasks.database_configuration || {}
+    if ActiveRecord::Base.configurations.empty?
+      ActiveRecord::Base.configurations = ActiveRecord::Tasks::DatabaseTasks.database_configuration
```

Comment:
Actually I do wonder if this line is still relevant considering that the databases rake task is under `railties`. We also have multiple spots in the rake task that references Rails.

Reply:
The tasks the reference Rails are only multiple database tasks. Since we need Railties to parse the database configuration to get the task names we decided that if you're using Active Record outside of a Rails application that we cannot support multiple database tasks.

Reply:
Thank you for clarifying :+1: We’ll continue to set the configurations the DatabaskTasks class then.